### PR TITLE
[CMAKE][CRT] Fix a few minor copypastas

### PR DESCRIPTION
**Pull request by SergeGautherie**
Be explicit...

### CODE REVIEW

File: sdk/lib/crt/math/exp2.c [hunk 1]
```diff
@@ -8,6 +8,6 @@ exp2(
     _In_ double x)
 {
     /* This below avoids clang to optimize our pow call to exp2 */
-    static const TWO = 2.0;
+    static const double TWO = 2.0;
```
Comment (learn-more) [inline sdk/lib/crt/math/exp2.c:11]: Does this still avoid clang to optimize the call?
Why did you change this?

Comment (ThFabba) [reply sdk/lib/crt/math/exp2.c:11]: To be honest, I don't know how the old version even compiles. It's not like types are optional in C.
(but yes, please verify that the generated assembly code is the same on all compilers)

Comment (ThFabba) [reply sdk/lib/crt/math/exp2.c:11]: Spoiler alert: this change does change the generated code.
To be fair, the new code looks better... but the gist is, this needs a test.

Comment (HBelusca) [reply sdk/lib/crt/math/exp2.c:11]: missing type?

**Review by learn-more — DISMISSED** [review]
Please show that the functional change is correct, and does not break what the workaround aimed to fix.

File: sdk/lib/crt/math/exp2.c (hunk 1 above)
Comment (zefklop) [reply sdk/lib/crt/math/exp2.c:11]: Oh, my bad :-( I don't even know how I could have missed that.

**Comment by zefklop** [comment]
> Please show that the functional change is correct, and does not break what the workaround aimed to fix.

The fact that clang build is not broken is already a good sign in itself. This on can go in, as far as I'm concerned.

**Review by zefklop — APPROVED** [review]

File: sdk/lib/crt/math/exp2.c (hunk 1 above)
Comment (zefklop) [reply sdk/lib/crt/math/exp2.c:11]: > Oh, my bad :-( I don't even know how I could have missed that.

-wImplicit is enabled by -Wall in both GCC & clang builds. But only clang actually uses this file, and -Werror is disabled here, that's how things slip through.

**Review by binarymaster — CHANGES_REQUESTED** [review]
Please resolve conflicting file.

image
